File: comparision/fib.py
import sys

def fib(n):
    if n<2:
        return 1
    else:
        return fib(n-1) + fib(n-2)

n = sys.argv[1] if len(sys.argv) >= 2 else 13
print 'fib(%d)=%d' % (int(n), fib(int(n)))

File: comparision/fib.pl
#!/usr/bin/env perl
use strict;
use warnings;
use utf8;
use 5.010000;
use autodie;

sub fib {
    my $n = shift;
    $n < 2 ? 1 : fib($n-1)+fib($n-2)
}

my $n = @ARGV > 0 ? $ARGV[0] : 13;
printf "fib(%d)=%d\n", $n, fib($n);

